src/main/java/org/tugraz/sysds/runtime/data/HomogTensor.java

```diff
@@ -0,0 +1,712 @@
+/*
+ * Copyright 2018 Graz University of Technology
```

Comment:
Update to 2019 license.

Reply:
Also did we loose the history again on this rename?

Reply:
I actually don't know I could keep the history then, I split the rename (`TensorBlock` to `HomogTensor`) and the implementation of the new `TensorBlock` into two commits on purpose so it would not interpret the new `TensorBlock` as the old one, but it seems even that didn't help.